### DISCUSSION TITLE
"Эксперты по боту" не показывает доступный для настройки сервер отдельным пользователям

### DISCUSSION THREAD

**Discussion by rovius** [LacunaHub/.github#28] [Общее]
При указании идентификатора отдельного пользователя в разделе "Эксперты по боту" этот пользователь не видит в профиле сервер, где ему выданы полномочия изменять настройки. Он может получить доступ, только если самостоятельно введёт в адресную строку `https://lacunabot.com/guilds/GUILD_ID/settings`. То же самое происходит, если указать не идентификатор, а роль без прав администратора и выдать её пользователям.

https://github.com/user-attachments/assets/a1f35288-2a8d-447e-81fd-3cd90d0d3bd3

**Comment by danliyev** [comment]
Пока нет возможности это исправить, т.к. при получении списка гильдий необходимо будет искать запись в базе данных для каждого сервера + получать информацию об участнике на сервере. Проблемы начнутся, если у пользователя будет много сообществ, где он состоит
